experimental/WreathMacdonaldpols/src/WreathMacdonaldpols.jl

```diff
@@ -0,0 +1,265 @@
+# I would like to credit Dario Mathiä who produced an initial version of the following code
+
+import Chevie: complex_reflection_group, charinfo, CharTable, CyclotomicNumbers
+export wreath_macs
+
+# Tools
+
+#gives indices of a subarray
+function subarray_indices(sub,arr)
+  res = Int[]
+  for i in 1:length(sub)
+    push!(res,findfirst(x->x==sub[i],arr))
+  end
+  return res
```

Comment:
Your code is perfectly fine, but just in case you are interested, a more idiomatic version would be
```suggestion
  return [findfirst(x->x==s, arr) for s in sub]
```
or even
```suggestion
  return [findfirst(==(s), arr) for s in sub]
```